runtime/cere-dev/src/lib.rs

```diff
@@ -1401,7 +1403,12 @@ pub type SignedPayload = generic::SignedPayload<RuntimeCall, SignedExtra>;
 pub type CheckedExtrinsic = generic::CheckedExtrinsic<AccountId, RuntimeCall, SignedExtra>;
 
 /// Runtime migrations
-type Migrations = ();
+type Migrations = (
+	pallet_ddc_clusters::migrations::v3::MigrateToV3<Runtime>,
+	pallet_ddc_clusters::migrations::v2::MigrateToV2<Runtime>,
+	pallet_ddc_staking::migrations::v1::MigrateToV1<Runtime>,
```

Comment:
`pallet_ddc_staking::migrations::v1` should be removed from `runtime/cere-dev/src/lib.rs` as it has been executed in Devnet already

Reply:
<img width="1438" alt="image" src="https://github.com/user-attachments/assets/12d1f9c1-c8fb-43b0-bfd5-b939ac37fed9">